nginx-opentracing-datadog.patch

```diff
@@ -1,186 +0,0 @@
-diff --git a/datadog/config b/datadog/config
```

Comment:
I'll try to remember this if I need a similar short-term trick in the future.
But also glad that term is over.